tests/connection_utils.py

```diff
@@ -15,21 +15,20 @@
 from chia.types.blockchain_format.sized_bytes import bytes32
 from chia.types.peer_info import PeerInfo
 from chia.util.ints import uint16
-from tests.setup_nodes import self_hostname
 from tests.time_out_assert import time_out_assert
 
 log = logging.getLogger(__name__)
 
 
-async def disconnect_all_and_reconnect(server: ChiaServer, reconnect_to: ChiaServer) -> bool:
+async def disconnect_all_and_reconnect(server: ChiaServer, reconnect_to: ChiaServer, self_hostname: str) -> bool:
     cons = list(server.all_connections.values())[:]
     for con in cons:
         await con.close()
     return await server.start_client(PeerInfo(self_hostname, uint16(reconnect_to._port)), None)
```

Comment:
does the hostname we claim to have matter for any test?
It might not, in which case we might be able to just pick a random string here and be done with it.

Reply:
I'm experimenting with simplifying this here: https://github.com/Chia-Network/chia-blockchain/pull/10371

Reply:
Ya, I initially had it that way with the global, too. Thanks!